src/shared-components/alert/test.tsx

```diff
@@ -14,97 +15,81 @@ const CustomContentComponent = () => (
   </div>
 );
 
-const createNodeMock = (element: ReactElement) => {
```

Comment:
As evidenced by the minimal snapshot changes, this workaround is no longer required.